sre-operational-fabric.md

```diff
@@ -0,0 +1,119 @@
+# (Remote) SRE - Rackspace Private Cloud
+
+## Role
+
+Site Reliability Engineering (SRE) is an engineering discipline that combines
+software and systems engineering to build and run large-scale, massively
+distributed, fault-tolerant systems. SRE ensures that Rackspace's managed
+service offerings & customer deployments have reliability and uptime
+appropriate to users' needs and a fast rate of improvement while monitoring and
+validating capacity and performance. Act as a Technical Lead for a portfolio of
+managed service deployments. Focus on reliability, scalability and the
+development of automation to manage a set of repetitive tasks at scale.
+
+### Knowledge, Skills, Ability
+
+- Deep Experience in one or more of: C, C++, Java, Perl, Python, Bash or Go.
+- Deep experience working with Unix/Linux systems from kernel to shell and beyond, with experience working with system libraries, file systems, and client-server protocols.
+- Networking: e.g. TCP/IP, UDP, ICMP, etc., MAC addresses, IP packets, DNS, SDN, OSI layers, and load balancing.
+- Strong expertise in designing, analyzing and troubleshooting large-scale distributed systems.
+- In depth knowledge of operating systems.
+- Familiarity with algorithms, data structures and complexity analysis.
+- Deep experience designing complex SaaS applications for cloud reliability and scalability.
+- Expert-level experience with GCP, AWS or Openstack APIs.
+- Deep experience with cloud infrastructure automation and CI/CD pipeline design.
+- Expertise in operational monitoring and management tools (Nagios, Datadog, etc.).
+- Advanced written & verbal communication skills, both highly technical and non-technical.
+- Ability to work closely with non-technical stakeholders and executives.
+- Capable of providing strategic technical advice.
+- Systematic problem-solving approach, coupled with a strong sense of ownership and drive.
+- Additional skills may be required depending on role; for example Kubernetes, Docker, Terraform, CEPH and other modern tools/technologies.
```

Comment:
Ah, there's Kubernetes. If you apply my above notes about Kube/Docker/Terraform, you could then update this bullet to something like "Domain-specific subject matter expertise a plus (Ceph, etc.)" or something.

Also: `CEPH` -> `Ceph`.